### DISCUSSION TITLE
`vendor:publish` is too difficult to read

### DISCUSSION THREAD

**Discussion by binotaliu** [laravel/framework#43771] [Ideas] [answered]
- Laravel Version: `9.25.1`
- PHP Version: `8.1.8`
- Database Driver & Version: `N/A`

### Description:
<img width="1062" alt="CleanShot 2022-08-19 at 16 50 57@2x" src="https://user-images.githubusercontent.com/67255597/185582281-f17ef466-cca2-4a9b-8c72-a15351077921.png">

The `choice` view is very hard to read now. While it looks cleaner than the old one, it's very difficult to tell which tag is corresponding to which ID.

The above image is the output from `php artisan vendor:publish` . What should I type if I want to choose `Tag: sail`? Is it 37, 38, or 39?

Now I need to use my mouse to select the line so I can make sure that I won't choose a wrong choice:
<img width="1079" alt="CleanShot 2022-08-19 at 17 00 26@2x" src="https://user-images.githubusercontent.com/67255597/185584231-1a99f389-1424-4fef-81f8-6ac02f319283.png">

I know I can use `--tag=` option, but it would be nice if we can just provide a better look on it. Not every one has good eyes. 

### Steps To Reproduce:

```
php artisan vendor:publish
```

**Comment by binotaliu** [comment]
Hi @nunomaduro, I'm not sure if it is ok to ping you. But can you take a look on this?

**Comment by Kamandlou** [comment]
I think it's good

**Comment by nunomaduro** [comment, marked as answer]
We are going to see ways to improve this. Thanks!
